src/style.css

```
@@ -1,3 +1,76 @@
-body {
+:root {
+  --blue-color: #00ACDC;
+}
+
+* {
   margin: 0;
 }
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+header {
```

Use the classes selector here
```suggestion
.header {
```